packages/components/src/FormatDate/FormatDate.tsx

```diff
@@ -8,9 +8,23 @@ interface FormatDateProps {
    * A `string` should be an ISO 8601 format date string.
    */
   readonly date: CivilDate | Date | string;
+
+  /**
+   * Boolean to show year or not.
+   */
+  readonly showYear?: boolean;
 }
 
-export function FormatDate({ date: inputDate }: FormatDateProps) {
+interface formatOptions {
+  readonly month: "short";
+  readonly day: "numeric";
+  readonly year?: "numeric";
+}
```

Comment:
```suggestion

```

Reply:
Good suggestion made the change [here](https://github.com/GetJobber/atlantis/pull/1301/commits/c382f2be8df523efafb2e64af73f710b9cc27291)